packages/sync/src/Listener.php

```diff
@@ -211,14 +211,6 @@ public function action_handler( ...$args ) {
 	public function bulk_enqueue_full_sync_actions( $action_name, $args_array ) {
 		$queue = $this->get_full_sync_queue();
 
-		/*
-		 * Periodically check the size of the queue, and disable adding to it if
-		 * it exceeds some limit AND the oldest item exceeds the age limit (i.e. sending has stopped).
-		 */
-		if ( ! $this->can_add_to_queue( $queue ) ) {
```

Comment:
Won't this be necessary for all the other modules that rely on this as a stopgap? See usage of `enqueue_all_ids_as_action()` in any module for example.

Reply:
`can_add_to_queue` checks to thing, if we have reached `max_queue_size` or the queue `lag`.
On `Full Sync` queue lag doesn't make much sense as we really don't care how old are those actions. We are still checking `max_queue_size` limits here: https://github.com/Automattic/jetpack/pull/13954/files/22c92ee9c3822babe91dcea2fdd565f8b2303be5#diff-09bb5fefa791ce30d5a0098d9fd19b86R271